### PR TITLE
Change ainEntityOfPage for Muziekschatten and Uitvoeringsmedium

### PR DESCRIPTION
**Pull request by rschalkrce**
Asked by data owner to change the `mainEntityOfPage` for these three datasets

### CODE REVIEW

File: packages/network-of-terms-catalog/catalog/datasets/muziekschatten-onderwerpen.jsonld [hunk 1]
```diff
@@ -22,7 +22,7 @@
     }
   ],
   "url": [
-    "https://data.muziekschatten.nl/som/"
+    "https://data.muziekschatten.nl/"
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/muziekschatten-onderwerpen.jsonld:25]: Red alert: does this mean URIs have changed? Or just that _new_ concepts may not have `/som/` in their URL?

File: packages/network-of-terms-catalog/catalog/datasets/uitvoeringsmedium.jsonld [hunk 2]
```diff
@@ -22,7 +22,7 @@
     }
   ],
   "url": [
-    "https://data.muziekschatten.nl/som/um"
+    "https://data.muziekschatten.nl/som/"
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/uitvoeringsmedium.jsonld:25]: So Uitvoeringsmedium still has `/som/` but subjects do not?

**Comment by rschalkrce** [comment]
Woops, my mistake! It should only concern `MainEntityOf`, so the dataset landing pages for Muziekschatten and Uitvoeringsmedium.

**Comment by ddeboer** [comment]
Fixed.

**Pull request merged**